src/webchat-ui/components/plugins/MessagePluginRenderer.tsx

```diff
@@ -34,11 +33,28 @@ export default ({ message, config, onSendMessage, plugins, isFullscreen, onSetFu
         : undefined;
 
     const matchedPlugins = getPluginsForMessage(plugins)(message);
+    const source = message.source;
+    const align = (source === 'bot' || source === 'agent') ? 'left' : 'right'; 
```

Comment:
i think it makes sense to also put this into the switch now. one less complicated inline-statement :D

Reply:
fixed